docs/api/v2/node.md

```diff
@@ -74,6 +74,16 @@ proc info*(node: WakuNode): WakuInfo =
   ##
```

Comment:
Missing comment and brief description at top of file:


    Init - create a node.
    Start - start a created node.
    Subscribe - to a topic or a specific content filter.
    Unsubscribe - to a topic or a specific content filter.
    Publish - to a topic, or a topic and a specific content filter.
    Query - for historical messages.
    Info - to get information about the node.

Reply:
Sure, done!

Reply:
@oskarth Let me know if further updates are needed.